contracts/bprotocol/avatar/AbsCToken.sol

```diff
@@ -23,7 +24,7 @@ contract AbsCToken is Cushion {
         return bComptroller.isBToken(bToken);
     }
 
-    function borrowBalanceCurrent(ICToken cToken) public returns (uint256) {
+    function borrowBalanceCurrent(ICToken cToken) public onlyBToken returns (uint256) {
```

Comment:
why did you add this modifier?

Reply:
on the Avatar side function takes `cToken` as an argument and on the BToken side, it takes `bToken` as an argument. So to avoid the confusion hiding `borrowBalanceCurrent` function on Avatar contract. Only exposing it to the user on BToken side.